Python/Product/TestAdapter/TestAnalyzer.cs

```diff
@@ -157,13 +160,26 @@ LocationInfo sourceLocation
             }
         }
 
+        /// <summary>
+        /// Get Test Case Members for a class.  If the class has 'test*' tests 
+        /// return those.  If there aren't any 'test*' tests return (if one at 
+        /// all) the runTest overridden method
+        /// </summary>
         private static IEnumerable<KeyValuePair<string, IAnalysisSet>> GetTestCaseMembers(
             IPythonProjectEntry entry,
             AnalysisValue classValue
         ) {
-            return classValue.GetAllMembers(entry.Analysis.InterpreterContext)
-                .Where(v => v.Key.StartsWith("test"))
+            var methodFunctions = classValue.GetAllMembers(entry.Analysis.InterpreterContext)
                 .Where(v => v.Value.Any(m => m.MemberType == PythonMemberType.Function || m.MemberType == PythonMemberType.Method));
+
+            var tests = methodFunctions.Where(v => v.Key.StartsWith("test"));

```

Comment:
Is it illegal to have both?  What happens if we find something with testSupport but they are using runTest